src/main/scala/nl/knaw/dans/lib/logging/servlet/body/LogResponseBody.scala

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (C) 2016 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.lib.logging.servlet.body
+
+import javax.servlet.http.HttpServletResponse
+import nl.knaw.dans.lib.logging.servlet.ResponseLogFormatter
+import org.scalatra.{ ActionResult, ScalatraBase }
+
+private[servlet] trait LogResponseBody extends ResponseLogFormatter {
```

Comment:
Why not also request bodies? For example deposit status changes are supposed to be small. You might have to check the size as the actual content might be out of our control.

Or any custom content like for example the number of uploaded files.

Reply:
@jo-pol and I decided not to log the request body. If you want to see stuff from the request body, add your own logs once you parsed the data.